react/features/base/dialog/components/native/ConfirmDialog.js

```diff
@@ -43,7 +43,7 @@ class ConfirmDialog extends BaseSubmitDialog<Props, *> {
      * @returns {string}
      */
     _getSubmitButtonKey() {
-        return 'dialog.confirmYes';
+        return this.props.okTitleKey || 'dialog.confirmYes';
```

Comment:
the naming here is a bit confusing, the method is called 'getSubmitButtonKey', which then returns 'this.props.okTitleKey' ? why "ok title" ? the word "title" does not fit in my opinion, I know you're pulling it from abstract class, but maybe we could have it adjusted ?

Reply:
To be honest, I think the name of ```_getSubmitButtonKey``` is misleading here. What do you think about ```_getConfirmButtonKey```?

Reply:
What I find confusing is the name of the property, since when buttons have titles :P ? especially in a dialog context Im thinking that tile is something that is displayed as dialog's title on the top of it. The method itself is okay

Reply:
Well, I agree about the "title" concern... Do you want me to change all the 24 occurrences of ```okTitleKey``` and ```cancelTitleKey```?

Reply:
I'd be grateful if you could do that, probably in a separate commit

Reply:
Ok! How about ```okKey``` and ```cancelKey```?

Reply:
sounds good to me !

Reply:
Done, it wasn't that much :)

Reply:
Thanks ! I will give it a try and then eventually approve later today